nuget/framework/nunit.nuspec

```diff
@@ -30,7 +30,6 @@ Supported platforms:
     <dependencies>
       <group targetFramework="netstandard1.6">
         <dependency id="NETStandard.Library" version="1.6.0" />
-        <dependency id="Microsoft.NETCore.Portable.Compatibility" version="1.0.1" />
```

Comment:
This dependency is also in the nunitlite nuspec - should that be cleaned up at the same time? The NUnitLite portable targets a slightly different profile to the nunit package, perhaps they could be matched up too.